packages/manager/cypress/e2e/core/kubernetes/lke-create.spec.ts

```diff
@@ -870,8 +874,12 @@ describe('LKE Cluster Creation with LKE-E', () => {
           capabilities: ['Kubernetes Enterprise'],
         })
       ).as('getAccount');
+      mockGetTieredKubernetesVersions('enterprise', [
+        latestEnterpriseTierKubernetesVersion,
+      ]).as('getTieredKubernetesVersions');
```

Comment:
```suggestion
      mockGetTieredKubernetesVersions('enterprise', [
        latestEnterpriseTierKubernetesVersion,
      ]).as('getTieredKubernetesVersions');
      mockGetTieredKubernetesVersions('standard', []);
```
I wonder if there'd be any benefit to mocking the `standard` versions response so Cloud doesn't try to send repeated requests to that endpoint during the test?

It doesn't seem to make a difference when it comes to the test failure I mentioned, though 😕